### PR TITLE
Feature/gui

### PR DESCRIPTION
**Pull request by yume-ind**
This PR creates the basic files for the Cassy gui, starting with the main VueClusters page. You can see some information about how to run the dev server by checking the `gui/README.md`

### CODE REVIEW

File: entrypoint.go [hunk 1]
```diff
@@ -14,9 +14,11 @@ import (
 
 var (
 	endpoint = flag.String("cassy server endpoint", "localhost:20051", "endpoint of Cassy server")
+	mode     = flag.Bool("dev", false, "enable this option to run as dev server")
```
Comment (Torch3333) [inline entrypoint.go]: Should this var name be more explicit like `devMode` or something like this ?

**Review by Torch3333 — COMMENTED** [review]
So far so good. 
Regarding the menu button, I am wondering if it won't be more practical to move it the left side.
At the moment, you click on the button which is on the right but it displays the contents on the left so I suppose this can be improve. What do you think ?

File: entrypoint.go (hunk 1 above)
Comment (yume-ind) [reply entrypoint.go]: Ok, I can change the variable name.

**Comment by yume-ind** [comment]
@Torch3333 I changed the navbar position and fixed the variable. Also, I realized I had forgotten to add a Register Cluster button on the page, so I added that and changed the style of the other buttons.

**Comment by Torch3333** [comment]
Thanks for the changes. I think you can continue the development.
We will see with Yamada-san tomorrow morning if he wants to see the UI before everything is finished or not.

**Pull request merged**